dashboard/app/controllers/levels_controller.rb

```diff
@@ -330,6 +330,7 @@ def level_params
     # Reference links should be stored as an array.
     if params[:level][:reference_links].is_a? String
       params[:level][:reference_links] = params[:level][:reference_links].split("\r\n")
+      params[:level][:reference_links].delete_if {|link| link == ""}
```

Comment:
using the `.blank?` ruby method here may be helpful:

```suggestion
      params[:level][:reference_links].delete_if {|link| link.blank?}
```

[here](https://railsless.blogspot.com/2011/08/difference-between-nil-empty-blank.html) is more info about ruby's .blank?, .present?, .nil?, etc method

Reply:
ooh -- then you can further shorten it to `.delete_if(&:blank?)`